Makefile

```
@@ -15,4 +15,5 @@ migrate_test:
 	docker compose run --rm app bash -c "goose -dir goose postgres 'postgres://user:pass@test_db/db?sslmode=disable' up"
 
 new_migration:
-	docker compose run --rm app bash -c "goose -dir goose postgres 'postgres://user:pass@db/db?sslmode=disable' create change_me sql"
\ No newline at end of file
+	docker compose run --rm app bash -c "goose -dir goose postgres 'postgres://user:pass@db/db?sslmode=disable' create change_me sql"
+	
```

Still missing the final line termination in this file.
```suggestion
	docker compose run --rm app bash -c "goose -dir goose postgres 'postgres://user:pass@db/db?sslmode=disable' create change_me sql"

```

---

I'll add that back